examples/bri-3/src/bri/auth/agent/auth.agent.ts

```diff
@@ -29,4 +30,17 @@ export class AuthAgent {
     const access_token = this.jwtService.sign(payload);
     return { access_token };
   }
+
+  async generateDidJwt(payload: any) {
+    const serviceDid = process.env.GOERLI_SERVICE_DID;
+    const privateKey = process.env.GEORLI_SERVICE_SIGNER_PRIVATE_KEY;
+    const serviceSigner = ES256KSigner(hexToBytes(privateKey));
+    const access_token = await createJWT(
```

Comment:
would be good to handle the cases if env variables are either empty or undefined

Reply:
we can, but since they pretty much must be defined, and cant be defaulted (for example private key can not have fallback value) all methods using them will throw exception and methods calling this will catch
is there some way you suggest we handle this?

Reply:
currently we are not assigning the field types to string, and hence TS doesn't complain.
But, if you do something like `const serviceDid : string = process.env.GOERLI_SERVICE_DID;` TS may complain that the value should be a string.

And yes, you have rightly mentioned that any functions using the undefined values will throw exception, that's why we should handle them all.

Reply:
let's discuss error handling in separate issue, current i see bunch of agents can throw exceptions but i dont see how we are catching them in command handlers
cc: @ognjenkurtic i forgot but i think we discussed this